server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -210,6 +231,14 @@ private String convertToString(ArrayList<String> comments) {
 
     for (String comment : comments) {
       comment = comment.replace("\"", "");
+      // This does makes sure of two things:
+      // 1. commment.substring(comment.length() - 1)
+      //    doesn't throw an error, which ruins the loop
+      // 2. One letter comments should not be considered as sentences since
+      //    they bring no value to the analysis. 
```

Comment:
I'd just stick with this comment. The one before is nice, but it's implied that empty values should be ignored normally,.